index.html

```diff
@@ -319,7 +319,6 @@ <h4>Let's chat!</h4>
                 </p>
```

Comment:
Please add linter for HTML, CSS, and JavaScript from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter config.